packages/account/src/Components/api-token/__tests__/api-token-article.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import ApiTokenArticle from '../api-token-article';
+
+it('should render ApiTokenArticle', () => {
+    render(<ApiTokenArticle />);
+    expect(screen.getByText('API token'));
+    expect(
+        screen.getByText(
+            /To access your mobile apps and other third-party apps, you'll first need to generate an API token./i
```

Comment:
```suggestion
            "To access your mobile apps and other third-party apps, you'll first need to generate an API token."
```

In one of your tests you used string format and here you used this format, IMO it's better to have one of these format and using that everywhere